react-props/my-app/src/App.tsx

```diff
@@ -0,0 +1,31 @@
+import { Header } from './Header';
+import { Image } from './Image';
+import { Caption } from './Caption';
+import { Description } from './Description';
+import { Button } from './Button';
+import './App.css';
+
+const imageVar = '/starry-sky.jpeg';
```

Comment:
I prefer to not use suffixes in variable names. They tend to just get in the way. I would prefer something more descriptive, like `imageSrc`, `caption`, `buttonLabel`, etc.